### ISSUE TITLE
How to package the app for distribution (Windows)

### ISSUE THREAD

**Issue by rfgamaral** [wavebox/waveboxapp#870]
As I've mentioned on other issues, I'm trying to build my own Wavebox version and I've been successful to an extent. I can't seem to package the built version into an .asar/.exe files and/or an installer. Would you mind sharing which tools and commands are you using to build that (as I believe those are not available as `pakcage.json` scripts?

Like #868, I'm just trying to build my own custom Wavebox version so that I can easily install on my system and run it with `wavebox.exe` instead of `electron bin/app/index.js`.

Can you give me a few hints?

**Comment by Thomas101** [comment]
Internally we use [electron-packager](https://www.npmjs.com/package/electron-packager) to build the binaries which has been setup for our internal use and is tied into our test suite, signing & distribution system etc.

For individual use, if you're playing around with custom css etc and just want to make a few tweaks I can see why it would be useful. I've just put a commit into git that will allow you to run the following, that creates a shortcut to run the app...

```
npm run create:runshortcut -- --out="C:/Users/Tom/Desktop"
```

...make sure you change the path for `out`. Clicking the shortcut icon will run `npm run dev:run` detached from the terminal, so you can play with any local changes you've got without the need to keep the terminal running.

**Comment by rfgamaral** [comment]
I appreciate that but what I'm really after is building my own binaries.

I've tried to use `electron-packager` but I'm not sure I'm using it properly, it takes a lot of time and doesn't actually build anything nor any error message is thrown. Maybe there's something wrong with my dev environment.

I tried to run it with something like `npx electron-packager . --out=dist` and it defaults to `--platform=win32` and `--arch=x64` (which is the binary I want to build). But it's taking hours and doesn't build anything.
